src/jit/gentree.cpp

```diff
@@ -14397,7 +14397,7 @@ bool Compiler::gtTreeHasSideEffects(GenTreePtr tree, unsigned flags /* = GTF_SID
     return true;
 }
 
-GenTreePtr Compiler::gtBuildCommaList(GenTreePtr list, GenTreePtr expr)
+GenTreePtr Compiler::gtBuildCommaList(GenTreePtr list, GenTreePtr expr DEBUGARG(bool remorphing))
```

Comment:
> perhaps I'm not seeing the value of preserving it in a subset of remorphing cases.

If we decide to make some transformations in morph value-number-preserving, the stricter check may be useful. As I mentioned, if this isn't super-compelling, I'll change the approach.